stl/src/sharedmutex.cpp

```diff
@@ -36,4 +36,9 @@ void __cdecl _Smtx_unlock_exclusive(_Smtx_t* smtx) { // unlock exclusive shared
 void __cdecl _Smtx_unlock_shared(_Smtx_t* smtx) { // unlock non-exclusive shared mutex
     ReleaseSRWLockShared(reinterpret_cast<PSRWLOCK>(smtx));
 }
+
+void __cdecl _Thrd_sleep_for(const long long ns) { // attempt to suspend thread for at least `ns` nanoseconds
+    constexpr long _Nsec_per_msec = 1'000'000L;

```

Comment:
The rest of the file is non-_Ugly, see `smtx` and `ns` variables, so it should be `nsec_per_msec`